etiqet-core/src/main/java/com/neueda/etiqet/core/transport/AsyncTransport.java

```diff
@@ -0,0 +1,24 @@
+package com.neueda.etiqet.core.transport;
+
+import com.neueda.etiqet.core.common.exceptions.EtiqetException;
+import com.neueda.etiqet.core.message.cdr.Cdr;
+
+import java.time.Duration;
+import java.util.Optional;
+import java.util.function.Consumer;
+
+public interface AsyncTransport extends Transport {
```

Comment:
Maybe BrokerTransport is a better name for this since it refers to subscriptions, topics ... so looks like something very related to BrokerClient-> BrokerTransport dependency

Reply:
done...